src/backend/templates/partials/footer.hbs

```diff
@@ -28,11 +28,16 @@
          {{#if copyright}}
          {{#if copyright.licence}}
          <p>
-         <a href="{{{copyright.licence_url}}}"> <img src="{{{copyright.logo}}}"> </a>
-         &copy; {{copyright.year}}
+        <a href="{{{copyright.licence_url}}}"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a>
```

Comment:
@Lplenka Please use `compact_logo` from the API under `licence_details` for the licence logo image.
Check it out here https://eventyay.com/api/v1/events/6